api/client/accesslist/accesslist.go

```diff
@@ -61,10 +61,12 @@ func (c *Client) GetAccessLists(ctx context.Context) ([]*accesslist.AccessList,
 }
 
 // ListAccessLists returns a paginated list of access lists.
+// Deprecated: Use [Client.ListAccessListsV2] instead.
+// TODO (avatus): DELETE IN 21.0.0
 func (c *Client) ListAccessLists(ctx context.Context, pageSize int, nextToken string) ([]*accesslist.AccessList, string, error) {
-	resp, err := c.grpcClient.ListAccessLists(ctx, &accesslistv1.ListAccessListsRequest{
+	resp, err := c.grpcClient.ListAccessListsV2(ctx, &accesslistv1.ListAccessListsV2Request{
```

Comment:
This needs to either be reverted (probably) or needs a fallback to ListAccessLists (can get kinda weird with the pagination tokens).